workos/users.py

```diff
@@ -0,0 +1,54 @@
+import workos
+from workos.resources.list import WorkOSListResource
+from workos.resources.users import (
+    WorkOSUser,
+)
+from workos.utils.request import (
+    RequestHelper,
+    REQUEST_METHOD_POST,
+)
+from workos.utils.validation import validate_settings, USERS_MODULE
+
+USER_PATH = "users"
+
+RESPONSE_LIMIT = 10
+
+
+class Users(WorkOSListResource):
+    """Offers methods to assist in authenticating through the WorkOS SSO service."""
```

Comment:
Is this doc comment correct? It looks like it corresponds to SSO.

Reply:
No. I'll fix that. 👍